conf/distro/tb-base.conf

```diff
@@ -0,0 +1,9 @@
+require conf/distro/include/pce-distro.inc
+
+PREFERRED_PROVIDER_virtual/kernel = "linux-tb"
+PREFERRED_VERSION_linux-tb ?= "5.5%"
+
+DISTRO = "tb-base"
+DISTROOVERRIDES = "tb:tb-base"
+
+DISTRO_FEATURES_append = " virtualization xen"
```

Comment:
If we name it `-base` then probably it should not contain xen features? Similarly as in case of the meta-pcengines.